examples/transformations/tga.py

```diff
@@ -0,0 +1,71 @@
+# Copyright (c) 2018 The Harmonica Developers.
+# Distributed under the terms of the BSD 3-Clause License.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# This code is part of the Fatiando a Terra project (https://www.fatiando.org)
+#
+"""
+Total gradient amplitude of a regular grid
+===================================
```

Comment:
Match the underline to the length of the title
```suggestion
Total gradient amplitude of a regular grid
==========================================
```